apps/builder/app/builder/features/style-panel/sections/backgrounds/background-thumbnail.tsx

```diff
@@ -0,0 +1,89 @@
+import { Image as WebstudioImage, loaders } from "@webstudio-is/image";
+import { styled, theme } from "@webstudio-is/design-system";
+import type { StyleInfo } from "../../shared/style-info";
+import brokenImage from "~/shared/images/broken-image-placeholder.svg";
+import env from "~/shared/env";
+
+const Thumbnail = styled("div", {
+  width: theme.spacing[10],
+  height: theme.spacing[10],
+  backgroundImage: "linear-gradient(yellow, red)",
+});
+
+const NoneThumbnail = styled("div", {
+  width: theme.spacing[10],
+  height: theme.spacing[10],
+  background:
+    "repeating-conic-gradient(rgba(0,0,0,0.22) 0% 25%, transparent 0% 50%) 0% 33.33% / 40% 40%",
```

Comment:
do we have that gradient in the tokens eventually?

Reply:
This is used in Image Control, its Chessboard for transparent background. I used it here that instead of webflow we have `none` default layer. And it's mostly the same as transparent.